torch/nn/mobile/models.py

```diff
@@ -0,0 +1,196 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import torch
+import torch.nn as nn
+import torch.nn.mobile as mobile
+import torch.quantization.fuse_modules as fuse
+
+import torchvision
+
+def mobilenetv2():
```

Comment:
@ljk53 @suo @dzhulgakov Having separate mobile-specific models is not ideal.  Ideally we want to perform these transformations in a JIT pass.  Not sure how to proceed here.  Should we move forward with this temporarily or move forward with merging the underlying implementation but not "exposing" it for now through these modules.  The old code paths is not going to be very efficient as mentioned in comments above.